cmd/fetcher/main.go

```diff
@@ -226,12 +232,13 @@ func updateBufPluginFile(name string, newVersion string) error {
 	return os.WriteFile(name, newData, 0644)
 }
 
-func copyFile(src, dest string) error {
+func copyFile(src, dest string, prevVersion, newVersion string) error {
 	data, err := os.ReadFile(src)
 	if err != nil {
 		return err
 	}
-	return os.WriteFile(dest, data, 0644)
+	replaced := strings.ReplaceAll(string(data), strings.TrimPrefix(prevVersion, "v"), strings.TrimPrefix(newVersion, "v"))
```

Comment:
For now just does a naive replacement of the previous version with the new version across all files.

Reply:
Probably good enough too. This will do it for _all_ files, but that's probably fine.